google/cloud/security/scanner/scanner.py

```diff
@@ -24,26 +24,21 @@
       --rules <rules path> \\
       --engine_name <rule engine name>
 """
-
-
-import itertools
-import os
-import shutil
 import sys
 
-from datetime import datetime
 import gflags as flags
 
 from google.apputils import app
-from google.cloud.security.common.data_access import csv_writer
 from google.cloud.security.common.data_access import dao
-from google.cloud.security.common.data_access import violation_dao
 from google.cloud.security.common.data_access import errors as db_errors
 from google.cloud.security.common.util import file_loader
 from google.cloud.security.common.util import log_util
 from google.cloud.security.scanner.audit import engine_map as em
-from google.cloud.security.scanner.scanners import scanners_map as sm
-from google.cloud.security.notifier import notifier
+from google.cloud.security.scanner.scanners import bigquery_scanner
```

Comment:
it would be nice if we could somehow load these dynamically from the config. Is this eventually what we plan to do?